.github/workflows/actions/setup_dev_ubuntu/action.yaml

```
@@ -0,0 +1,84 @@
+name: 'Setup dev environment'
+description: 'Setup a dev environment for rgh-action Ubuntu'
```

What is rgh-action?

---

It does not exist, my bad  :)

```suggestion
description: 'Setup a dev environment for gh-action Ubuntu'
```